mobile/ios/Podfile.lock

```diff
@@ -387,6 +387,8 @@ PODS:
     - React-Core
   - RNCMaskedView (0.1.11):
     - React
+  - RNDeviceInfo (10.0.2):
+    - React-Core
```

Comment:
This has been removed from the codebase, the `pod install` shouldn't cause this to be added. Do you have the latest from `dev`?

Reply:
I do, but for some reason, pod install still causes these changes?